alacritty.yml

```diff
@@ -497,45 +501,79 @@
 #   - ScrollToTop
 #   - ScrollToBottom
 #   - ClearHistory
+#       Remove the terminal's scrollback history.
 #   - Hide
+#       Hide the Alacritty window.
 #   - Minimize
+#       Minimize the Alacritty window.
 #   - Quit
+#       Quit Alacritty
 #   - ToggleFullscreen
 #   - SpawnNewInstance
+#       Spawn a new instance of Alacritty that preserves the path. Only applies to Linux/BSD.
 #   - ClearLogNotice
+#       Clear Alacritty's UI warning and error notices.
 #   - ClearSelection
+#       Remove the active selection.
 #   - ReceiveChar
 #   - None
 #
 #   (`mode: Vi` only):
 #   - Open
+#       Attempt to open URL under the cursor. See also url.launcher.
 #   - Up
+#       Move the vi mode cursor up by one line.
 #   - Down
+#       Move one character down.
```

Comment:
As mentioned in the comment about the `Up` action, it applied to all motions. So please change these too.

Reply:
Thanks for catching that.

I missed those parallel changes last time. Fixed in my next commit.